src/js/themes/base.d.ts

```diff
@@ -663,6 +663,11 @@ export interface ThemeType {
       background?: BackgroundType;
       color?: ColorType;
       margin?: MarginType;
+      icon?: any;
+      iconContainer: {
```

Comment:
Should this be updated to have any box props?

Reply:
Or we will just highlight these main ones, but all are accepted

Reply:
oh yes let me check how other components are addressing this when you accept all props